### PR TITLE
fix: qr code for short url

### LINKED ISSUE #106: Short URL QR Code Not Working
### Issue Description
The short URL QR code is not working.

### Expected Behavior
The short URL QR code should successfully direct users to the intended URL when scanned.

### Current Behavior
Scanning the short URL QR code does not redirect to the intended URL.

### Screenshots
41a67

### Reproducibility

 - [x] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. Scan the short URL QR code
2. Observe the result and note the failure to redirect to the intended URL.

### Severity/Priority
 - [ ] Critical
 - [x] High
 - [ ] Medium
 - [ ] Low

### Additional Information
- The issue has been observed across multiple devices.
- The short URL itself works correctly when accessed directly in a browser.

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by sahsisunny**
## Issue Ticket Number

-   Closes #106 

## Description

Fixed the issue where the short URL QR code was not working. Updated the QR code generation process to ensure proper redirection to the intended URL.

**Breaking Changes**

-   [ ] Yes
-   [x] No

**Development Tested?**

-   [x] Yes
-   [ ] No

**Tested in Staging?**

-   [x] Yes
-   [ ] No

**Under Feature Flag**

-   [ ] Yes
-   [x] No

**Database Changes**

-   [ ] Yes
-   [x] No

##### BEFORE
41a67

#### AFTER
4cd88 (1)

## Test Coverage

- Manual testing of QR code scanning on multiple devices to ensure proper redirection.

image

### CODE REVIEW

File: src/components/QRCodeModal/index.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const QRCodeModal: React.FC<QRCodeModalProps> = ({ shortUrl, onClose }) => {
                         src: RDSIcon,
                         height: 50,
                         width: 50,
-                        excavate: true,
+                        excavate: false,
                     }}
                     renderAs="canvas"
                     level="L"
```
Comment (vinit717) [inline src/components/QRCodeModal/index.tsx:69]: Still when I run this locally I can't scan 
as QRcode is deprecated and removing image settings makes QR code work 
https://www.npmjs.com/package/qrcode.react#qrcode---deprecated

Comment (sahsisunny) [reply src/components/QRCodeModal/index.tsx]: Can you share the QR code image here?

Comment (sahsisunny) [reply src/components/QRCodeModal/index.tsx]: Removed RDS logo from QR code

Comment (vinit717) [reply src/components/QRCodeModal/index.tsx]: After removing logo? I have to send?
and also use QRCodeCanvas instead of QRCode  as QRCode  is deprecated

Comment (sahsisunny) [reply src/components/QRCodeModal/index.tsx]: Done

**Review by vinit717 — APPROVED** [review]

**Review by iamitprakash — CHANGES_REQUESTED** [review]
code is not working

**Comment by sahsisunny** [comment]
> code is not working

QR Code is working

> code is not working fine


https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/70854507/2401ca92-3c82-458d-93a8-4ee5f5d6b07a

**Comment by iamitprakash** [comment]
> > code is not working
> 
> QR Code is working
> 
> > code is not working fine
> 
>  11111.mp4
not working into my phone, so I will assume QR is not working

**Review by Achintya-Chatterjee — APPROVED** [review]

**Comment by sahsisunny** [comment]
TODO: Find the reason to excavate false to work QR Code

**Review by yesyash — APPROVED** [review]

**Review by Achintya-Chatterjee — DISMISSED** [review]

**Review by iamitprakash — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**